### ISSUE TITLE
Unable to deploy api with latest cli

### ISSUE THREAD

**Issue by robotrekker** [webiny/webiny-js#684]
## This is:

- Bug

## Specifications

  - version: 2.6.0
  - OS: MacOS
  - Browser: Chrome

## Expected Behavior
webiny deploy-api should perform a deploy for the local environment.

## Actual Behavior
Get the following error:
An error occurred during deployment of cognito
TypeError: this.context.instance.getResourceName is not a function

## Steps to Reproduce the Problem
1. Follow Quick Start steps
2. On webiny deploy-api error occurs

## Detailed Description
It worked for me yesterday (1/28/2020) with a different project.  Today I get the error mentioned.  Looked at the package.json and today it seems to include newer dependencies.

package.json (1/28)
"dependencies": {
		"@webiny/api-cookie-policy": "^2.1.1",
		"@webiny/api-files": "^2.2.1",
		"@webiny/api-forms": "^2.1.1",
		"@webiny/api-google-tag-manager": "^2.1.1",
		"@webiny/api-i18n": "^2.2.1",
		"@webiny/api-mailchimp": "^2.1.1",
		"@webiny/api-plugin-commodo-mongodb": "^1.1.1",
		"@webiny/api-plugin-create-apollo-gateway": "^1.1.0",
		"@webiny/api-plugin-create-apollo-handler": "^1.1.0",
		"@webiny/api-plugin-files-resolvers-mongodb": "^1.1.0",
		"@webiny/api-plugin-page-builder-resolvers-mongodb": "^1.1.0",
		"@webiny/api-plugin-form-builder-resolvers-mongodb": "^1.2.0",
		"@webiny/api-plugin-security-cognito": "^2.1.0",
		"@webiny/api-security": "^2.1.2",
		"@webiny/serverless-api-gateway": "^2.0.1",
		"@webiny/serverless-app": "^2.0.0",
		"@webiny/serverless-apollo-gateway": "^2.2.0",
		"@webiny/serverless-apollo-service": "^2.2.0",
		"@webiny/serverless-aws-cloudfront": "^2.0.0",
		"@webiny/serverless-aws-cognito-user-pool": "^2.1.0",
		"@webiny/serverless-files": "^2.3.3",
		"@webiny/serverless-page-builder": "^2.2.1",
		"@webiny/serverless-form-builder": "^2.0.3"
	},

package.json (1/29/2020 - today)
"dependencies": {
		"@webiny/api-cookie-policy": "^3.0.0",
		"@webiny/api-files": "^3.0.0",
		"@webiny/api-forms": "^2.1.1",
		"@webiny/api-google-tag-manager": "^3.0.0",
		"@webiny/api-i18n": "^3.0.0",
		"@webiny/api-mailchimp": "^3.0.0",
		"@webiny/api-plugin-commodo-mongodb": "^2.0.0",
		"@webiny/api-plugin-create-apollo-gateway": "^2.0.0",
		"@webiny/api-plugin-create-apollo-handler": "^2.0.0",
		"@webiny/api-plugin-files-resolvers-mongodb": "^2.0.0",
		"@webiny/api-plugin-page-builder-resolvers-mongodb": "^2.0.0",
		"@webiny/api-plugin-form-builder-resolvers-mongodb": "^1.2.0",
		"@webiny/api-plugin-security-cognito": "^3.0.0",
		"@webiny/api-security": "^3.0.0",
		"@webiny/serverless-api-gateway": "^3.0.0",
		"@webiny/serverless-app": "^3.0.0",
		"@webiny/serverless-apollo-gateway": "^3.0.0",
		"@webiny/serverless-apollo-service": "^3.0.0",
		"@webiny/serverless-aws-cloudfront": "^3.0.0",
		"@webiny/serverless-aws-cognito-user-pool": "^3.0.0",
		"@webiny/serverless-files": "^3.0.0",
		"@webiny/serverless-page-builder": "^3.0.0",
		"@webiny/serverless-form-builder": "^3.0.0"
	},

## Possible Solution
Was the serverless component to deploy cognito updated causing a breaking issue?

**Comment by Pavel910** [comment]
@robotrekker the CLI was updated to version 3, so you need to upgrade your global `@webiny/cli` package. And when you do, please create a new project, because there were breaking changes, and the project setup with the old CLI will not work.

**Comment by robotrekker** [comment]
Thx for the prompt response, you guys are awesome.  Love, love, love this project.  I did a global install and still after version check was not up to 3.0.0.  After digging deeper, it seems I had two versions installed somehow (probably yarn/npm mismatch issue).  I cleaned it up by removing both installs and reinstalled clean, verified the version and it is now working when I create a new app as mentioned by Pavel910.  Thanks again Pavel910 for taking the time to help me straighten this out.
